elpy.el

```diff
@@ -312,6 +312,19 @@ edited instead. Setting this variable to nil disables this feature."
   :type 'boolean
   :group 'elpy)
 
+(make-obsolete 'elpy-eldoc-show-current-function
+               "Please use `elpy-eldoc-info' instead" "1.25.0")
+
+(defcustom elpy-eldoc-info 'docstring
```

Comment:
I think we can live without this option, can't we? Just use fallbacks, calltip if available, else docstring if available, else current-function?